app/views/venues/_search.html.erb

```diff
@@ -1,7 +1,7 @@
 <li>
   <div class='row collapse search'>
     <%= form_tag venues_path, method: :get, class: 'search-form' do %>
-      <div class="large-4 columns">
+      <div class="large-8 medium-8 small-6 columns">
         <%= text_field_tag :query, nil, placeholder: "Search"%>
       </div>
```

Comment:
We have one, it's getting a `display:none` rule.  Hitting enter will submit the search query.